### PR TITLE
Add purchase order protobuf files

### PR DESCRIPTION
**Pull request by RyanLassigBanks**
Signed-off-by: Ryan Banks <rbanks@bitwise.io>

### CODE REVIEW

File: sdk/protos/purchase_order_payload.proto [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+message UpdateVersionPayload {
+  PurchaseOrderRevision revision = 1;
+}
+
+message PurchaseOrderRevision {
+  uint64 revision_number = 1;
+  string submitter = 2;
+  uint64 created_at = 3;
+
+  string order_xml_34 = 4;
```
Comment (peterschwarz) [inline sdk/protos/purchase_order_payload.proto:54]: missed a 34 -> v3_4 change

**Review by peterschwarz — APPROVED** [review]

File: sdk/protos/purchase_order_payload.proto [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+message CreateVersionPayload {
+  bool is_draft = 1;
+  PayloadRevision revision = 2;
+}
+
+message UpdateVersionPayload {
+  PayloadRevision revision = 1;
+}
+
+message PayloadRevision {
```
Comment (dnewh) [inline sdk/protos/purchase_order_payload.proto:50]: Can this just import the PurchaseOrderRevision from the state protos

**Review by dnewh — CHANGES_REQUESTED** [review]

File: sdk/protos/purchase_order_payload.proto (hunk 2 above)
Comment (vaporos) [reply sdk/protos/purchase_order_payload.proto:50]: No, there should be no overlap in state or transaction definitions as they represent separate APIs.

File: sdk/protos/purchase_order_payload.proto [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    CREATE_VERSION = 2;
+    UPDATE_VERSION = 3;
+  }
+  Action action = 1;
+  string org_id = 2;
+  string public_key = 3;
+  uint64 timestamp = 4;
+
+  CreatePurchaseOrderPayload create_po_payload = 5;
+  UpdatePurchaseOrderPayload update_po_pyaload = 6;
```
Comment (dnewh) [inline sdk/protos/purchase_order_payload.proto:31]: typo pyaload -> payload

**Review by dnewh — CHANGES_REQUESTED** [review]

File: sdk/protos/purchase_order_payload.proto (hunk 3 above)
Comment (RyanLassigBanks) [reply sdk/protos/purchase_order_payload.proto]: updated

File: sdk/protos/purchase_order_state.proto [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// -----------------------------------------------------------------------------
+
+syntax = "proto3";
+
+message PurchaseOrder {
+  string uid = 1;
+  string workflow_status = 2;
```
Comment (dnewh) [inline sdk/protos/purchase_order_state.proto:20]: This should have an `accepted_version_id` field

**Review by dnewh — CHANGES_REQUESTED** [review]

File: sdk/protos/purchase_order_state.proto (hunk 4 above)
Comment (RyanLassigBanks) [reply sdk/protos/purchase_order_state.proto:20]: updated

**Review by dnewh — DISMISSED** [review]

**Review by vaporos — APPROVED** [review]

**Review by shannynalayna — APPROVED** [review]

**Pull request merged**